libraries/base/base.cabal

```diff
@@ -3,7 +3,7 @@ version:        4.11.1.0
 license:        BSD3
 license-file:   LICENSE
 maintainer:     libraries@haskell.org
-bug-reports:    http://ghc.haskell.org/trac/ghc/newticket?component=libraries/base
+bug-reports:    https://gitlab.haskell.org/ghc/ghc/-/issues/new
```

Comment:
I'm not sure the url structure beyond `/new`. It's unfortunate that this information isn't current, I might have wanted to file bugs against these libraries.